Resources/Prototypes/Entities/Objects/Misc/paper.yml

```diff
@@ -7,14 +7,22 @@
   - type: Sprite
     sprite: Objects/Misc/bureaucracy.rsi
     layers:
-    - state: paper_words
     - state: paper
   - type: Paper
   - type: UserInterface
     interfaces:
     - key: enum.PaperUiKey.Key
       type: PaperBoundUserInterface
 
+- type: entity
+  parent: Paper
+  id: PaperWritten
+  abstract: true
+  components:
+  - type: Sprite
+    layers:
+    - state: paper_words
```

Comment:
im unsure about this, but please check it since i cant rn:
i think writing it like this causes the spritecomponent to inherit the previous layers, meaning the paperwritten prototype will now have two layers (paper, paper_words)

Reply:
Given that it *already* had two layers before I changed things, and now only has one layer, this isn't really worsening things, but I'll still check, hold on

Reply:
Nope, it only has one layer

Reply:
consider this resolved then